wp-includes/class-wp-service-workers.php

```diff
@@ -37,6 +37,14 @@ class WP_Service_Workers extends WP_Scripts {
 	 */
 	const SCOPE_ALL = 3;
 
+	/**
+	 * Stale while revalidate caching strategy.
+	 *
+	 * @todo Add more strategies.
+	 * @var int
+	 */
+	const STRATEGY_STALE_WHILE_REVALIDATE = 1;
```

Comment:
Instead of `1` I think better to just use the workbox identifiers, for example `'staleWhileRevalidate'`. That will make it easier.